MacOS.sh

```diff
@@ -18,7 +18,7 @@ read -p "Enter name of the computer object in Active Directory: " ADCOMPUTER
 read -p "Enter OU where the computer object will be created: " OU
 
 # Join computer to Active Directory
-sudo dsconfigad -add "$DOMAIN" \
+sudo dsco-OH-NO-IS-THIS-WRONG-??-figad -add "$DOMAIN" \
```

Comment:
The command `dsco-OH-NO-IS-THIS-WRONG-??-figad` appears to be a typo. Please verify the correct command for joining a computer to Active Directory.